SU-SWS/stanford_news
# D8 core 2052 - Visual QA edits

# READY FOR REVIEW

# Summary
- Ticket for the issues logged by design.

# Needed By (Date)
- May 13th

# Urgency
- Fairly high

# Steps to Test

1. Review code
2. Pull in branch.
3. Verify issues it ticket are correct.

# Affected Projects or Products
- D8CORE
- SOE & LelandD8

# Associated Issues and/or People
- News Accelerant

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
